index.html

```diff
@@ -5,16 +5,20 @@
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <link href="css/stylesheet.css" rel="stylesheet">
+    <link href="https://unpkg.com/leaflet@1.0.1/dist/leaflet.css" rel="stylesheet"/>
     <title>Geolocator for FOSSASIA</title>
 </head>
 <body>
     <h2>Geo locator to faciliate easy integration of location data</h2>
     <div id="main-div" class="card">
         <label class="label">Click the below button to get Geolocation</label><br><br><br>
-        <input id="button" type="button" value="Get my geolocation" onClick="geolocation()">
+        <input id="button" type="button" value="Get my geolocation" onclick="geolocation()">
         <br><br>
         <!-- The below div will hold the data -->
         <label id="output" class="label"></label>
+        <label id="preview" class="label"></label>
+        <div id="osm-map"></div>
+		<!--<iframe id="frameID" width="200" height="200"></iframe>-->
```

Comment:
Why is this commented out? Is it needed?

Reply:
No, It was the testing process, as some mentor said that you can use <iframe> to include the map, I tested but it was looking messy, so did via native osm api :blush:

Reply:
Should I remove the comment @m1guelpf ?

Reply:
It is a bad practice to keep debugging code or code from a previous implementation commented out in production...

Reply:
Got it!

Reply:
@m1guelpf Done :+1: 
Thanks for telling me this :blush: